ddtrace/internal/debug.py

```diff
@@ -55,6 +47,9 @@ def collect(tracer):
             agent_error = "Agent not reachable at %s. Exception raised: %s" % (agent_url, str(e))
         else:
             agent_error = None
+    else:
+        agent_url = "CUSTOM"
```

Comment:
`"UNKNOWN"`?

Not sure if "custom" makes sense here unless someone knows about this check/branch?

Reply:
hmm, seeing your test example below.

Reply:
These logs are just for us/support and in this context we just care about whether an agent is accessible or not which isn't applicable if a custom trace writer is being used.